wpilibc/src/main/native/include/frc/PWM.h

```diff
@@ -166,10 +166,10 @@ class PWM : public wpi::Sendable, public wpi::SendableHelper<PWM> {
   /**
    * Optionally eliminate the deadband from a speed controller.
    *
-   * @param eliminateDeadband If true, set the motor curve on the Jaguar to
-   *                          eliminate the deadband in the middle of the range.
-   *                          Otherwise, keep the full range without modifying
-   *                          any values.
+   * @param eliminateDeadband If true, set the motor curve on the speed
+   *                          controller to eliminate the deadband in the middle
```

Comment:
Didn't we rename SpeedController to MotorController? Should flip the wording here too whilst we're here.

Reply:
speed controller is used throughout this file. Given #3302, we probably should update this file to use motor controller, but that's affects a lot more then just this line.